cmd/parca-agent/main.go

```diff
@@ -927,6 +928,14 @@ func run(logger log.Logger, reg *prometheus.Registry, flags flags) error {
 		})
 	}
 
+	optimizedSymtabs := filepath.Join(flags.Debuginfo.TempDir, "optimized_symtabs")
+	if os.RemoveAll(optimizedSymtabs); err != nil {
```

Comment:
Did you mean?

```suggestion
	if err := os.RemoveAll(optimizedSymtabs); err != nil {
```

Reply:
oh wow, I didn't even expect this to compile

Reply:
(unrelated to the PR feedback) I prefer to avoid this pattern because I find it unreadable and it's too easy to make mistakes like this one. Perhaps a good opportunity for a new linter? 😄

Reply:
For future refrences, this wasn't caught in CI because golangci-lint excludes it https://github.com/golangci/golangci-lint/blob/855d443351417cfec74f1c832cab2c52ecedcb71/pkg/config/issues.go#L13-L14